src/macros.rs

```diff
@@ -150,7 +150,7 @@ macro_rules! libc_bitflags {
     // (non-pub) Entry rule.
     (
         $(#[$attr:meta])*
-        flags $BitFlags:ident: $T:ty {
+        pub flags $BitFlags:ident: $T:ty {
```

Comment:
I think this is a (or maybe even _the_) problem. The macro `libc_bitflags` already differentiated between pub and non-pub bitflags (as the comments indicate).